Packs/CommunityCommonScripts/Scripts/DeleteIndicators/DeleteIndicators.py

```diff
@@ -0,0 +1,45 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    try:
+        args = demisto.args()
+        query = args.get('indicator_query', None)
+        indicator_vals = args.get('indicator_values', None)
+        indicator_ids = args.get('indicator_ids', None)
```

Comment:
```suggestion
        indicator_vals = argToList(args.get('indicator_values', None))
        indicator_ids = argToList(args.get('indicator_ids', None))
```

Reply:
Using `argToList` ensures `indicator_values` and `indicator_ids` are always handled as lists, improving consistency and preventing errors.